pkg/registry/pod/etcd/etcd.go

```diff
@@ -142,15 +142,24 @@ func (r *BindingREST) New() runtime.Object {
 // Create ensures a pod is bound to a specific host.
 func (r *BindingREST) Create(ctx api.Context, obj runtime.Object) (out runtime.Object, err error) {
 	binding := obj.(*api.Binding)
-	err = r.assignPod(ctx, binding.PodID, binding.Host)
+	err = r.assignPod(ctx, binding.PodID, binding.Host, binding.Annotations)
 	err = etcderr.InterpretCreateError(err, "binding", "")
-	out = &api.Status{Status: api.StatusSuccess}
+	if err != nil {
```

Comment:
Hrm, this is somewhat wrong, because InterpretCreateError may already create an appropriate api.Status.  If assignPod needs to return special errors, that should happen in that method (using errors.NewConflict)